### ISSUE TITLE
Yesh and Brian Calls

### ISSUE THREAD

**Issue by Yesh-2691** [smartcontractresearchforum/operations#81]
### Aug 04, 2022
- [x] Multiple assigneees
- [x] What to use when? Projects Vs. Milestones? 
  - [x]  The way I understand it is - Projects has a process to it. Milestones are just fixed set of issues that when completed finishes the 
  milestone. 
  - [x]        How can we see an overview of Milestones?  - Manually maintaining the overview  - Brain to share the link later
  - [ ]       Do Milestones capture the correct 'project' information? 
  - [ ]         What if someone does not include issues into a Milestone?
  - [ ]         Is there the Classic Projects Notes section?

- [x] Can 'Views' be classified as projects?
  - [ ] Having different workflows within one project?

- [ ] What is up with the formatting in Github
- [ ] Can you tell me a bit about Epics?
- [x] Can you please give me admin access to the Ops Repo. Need it for creating the issue template
- [x] Visual studio code - to edit .yml file

**Comment by SirAh0** [comment]
**Standards**
Only issues that represent work should be added to a milestone. If an issue is informational only - it will never have a reasonable close state and thus skews the reported percent done on the milseton.

**Comment by Yesh-2691** [comment]
Aug 17, 2022

**Comment by SirAh0** [comment]
Thoughts on Issue Template via REST API
form submission via [scrf.io](http://scrf.io/) into an issue
- [tina.io](http://tina.io/) -> creates new md file (form submission?)
	- trigger action
	- action create issue

**Comment by SirAh0** [comment]
TODO
- [ ] Design pass on the [Project Proposal](https://docs.google.com/document/d/11jQwqa3iw4ZrYDz-idI4ZTUMKPqXPbE2LBEt2SEx2vo/edit#heading=h.k1pi75mo3yob)

**Comment by SirAh0** [comment]
**More Empty Design Templates**
- We have one for presentations
- We should also have some for other kinds of collateral
- An empty Doc with standard Header / Footer sections

**Comment by marvino3** [comment]
@SirAh0 @Yesh-2691 are we looking for a doc/ one pager template?
If so, we've got a tic for that - https://github.com/smartcontractresearchforum/discovery/issues/50
It was deprioritized because it wasn't a pressing need, but we can move it up if this is something we now want

Would we use the Project Proposal doc you linked above as an example? https://docs.google.com/document/d/11jQwqa3iw4ZrYDz-idI4ZTUMKPqXPbE2LBEt2SEx2vo/edit#

**Comment by Yesh-2691** [comment]
@marvino3 Yes we are. @SirAh0 introduced me to the Design templates used for the website. I think we can utilize that. Brian will try to match the design to the current proposal template and then run it by you for approval. 

I however think we need to have multiple blank templates ready with SCRF design. The documents rn look kinda dull without the logo and the colors. 

Regarding the proposal, we need to make a few final edits on that and then will be rolling it out during the next PPP or community call. It is more or less the final version. Please feel free to suggest any design recommendations in line with SCRF discovery.

**Comment by Yesh-2691** [comment]
Oct 28, 2022

- [x] Calling an issue from another repo.
- [x] Claiming bounties. 
- [x] Label as trigger points for issue transfer to bounties.
